src/browser/modules/D3Visualization/components/ResultsPane.tsx

```diff
@@ -79,38 +86,40 @@ export class LegendComponent extends Component<any, State> {
         return (
           <StyledLegendInlineListItem key={i} data-testid="viz-legend-labels">
             <StyledLegendContents className="contents">
-              <StyledLabelToken
-                onClick={onClick}
-                style={style}
-                className="token token-label"
+              <Popup
+                on="click"
+                basic
+                pinned
+                trigger={
+                  <StyledLabelToken
+                    onClick={onClick}
+                    style={style}
+                    className="token token-label"
+                  >
+                    {legendItemKey}
+                    <StyledTokenCount className="count">{`(${numberToUSLocale(
+                      labels[legendItemKey].count
+                    )})`}</StyledTokenCount>
+                  </StyledLabelToken>
+                }
+                inverted={this.props.theme === DARK_THEME}
```

Comment:
padding around grass editor in dark theme had a light padding so had to switch `inverted` property on here. There may be another way around this.